main.py

```diff
@@ -21,7 +24,23 @@
 
 PRODUCTION = False # added for convenience, all factories eventually created in main and passed to trade_bot
 
+# SERVER SETUP
+def flask_init():
+    endpoints = []
+
+    endpoints.append(Endpoint("/orders-open", get_open_orders))
+    endpoints.append(Endpoint("/orders-closed", get_closed_orders))
+
+    api = API(__name__, endpoints)
```

Comment:
API seems like a very general name for this class, I think calling it something like WebService() for the class & and its variable might be web_service might make more sense